eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -325,7 +325,7 @@ jobs:
 
     # Generate Documentation
     - pwsh: |
-        $setupFile="doxygen-1.8.18-setup.exe";
+        $setupFile="doxygen-1.8.20-setup.exe";
```

Comment:
Do we need to update Docs to mention what's the min Doxygen version that needs to be used to correctly generate docs for the project?

Reply:
Maybe. Although I don't think this particular version is necessary. Whatever reasonably new version folks install, will work, including 1.8.18.